samples/highcharts/sonification/big-data/demo.js

```diff
@@ -4,13 +4,24 @@ for (var i = 0; i < 1000; ++i) {
     data.push(Math.sin(i / 50) * 3 + Math.random() - 0.5);
 }
 
-Highcharts.chart('container', {
+Highcharts.setOptions({
+    lang: {
+        accessibility: {
+            chartContainerLabel: 'Click series to sonify chart.'
```

Comment:
Would replace this label with `"Big data sonification. Highcharts interactive chart."`